server/main.py

```diff
@@ -51,7 +52,7 @@ async def get_data(username:str,new:Get,response:Response):
     password = new.password
     username = username
     async with database.transaction():
-        await database.execute("create table if not exists user( id integer primary key autoincrement, username text not null, password text not null, key blob default None,cookie text unique default None) ;")
+        
         row = await database.fetch_one("select key from user where username=:username and password=:password;",{"username":username,"password":password})
```

Comment:
A transaction is not required if you are only doing single `fetch`.

As single queries are already "Atomic", i.e. there will be no other changes to database while this query is running. We need transaction when running multiple queries.


```python
async with database.transaction():
    row = await.database.fetch_one("select user from users where id=3 FOR UPDATE")
    # <---- transaction will make sure, that no query modifies `user` while this transaction is completed --------->
    await database.insert("insert into session_tokens ...")

# <---- once we are out of transaction database is `unlocked` again and other queries can modify it.
```